### DISCUSSION TITLE
组件与其他表格功能并存场景

### DISCUSSION THREAD

**Discussion by testBot-alt** [varown/react-antd-column-resize#22] [General]
![image](https://github.com/varown/react-antd-column-resize/assets/78595451/0e4561b2-535e-4fae-9791-f5bbc1529b19)
您好，我在项目中需要去做可编辑表格的功能，需要去配置components属性，可是这个属性会与该组件冲突，所以我想问下怎么在使用resize组件的同时应用可编辑表格，以下是可编辑表格的components配置
![image](https://github.com/varown/react-antd-column-resize/assets/78595451/9ae0415d-2619-4430-9bc8-4b8885cf79f5)

**Comment by testBot-alt** [comment]
https://blog.csdn.net/weixin_46533954/article/details/131935999
具体的可编辑表格的组件代码和以上文档中的代码差不多

**Comment by varown** [comment]
你好，你这边遇到了什么问题呢

**Comment by varown** [comment]
https://github.com/varown/react-antd-column-resize/blob/master/src/useAntdColumnResize/demo/edit.tsx
麻烦参考下这个示例  测试是没问题的

**Comment by varown** [comment]
https://varown.github.io/react-antd-column-resize/components/use-antd-column-resize#src-use-antd-column-resize-demo-edit
这里是可编辑拖动的演示你看下

**Comment by testBot-alt** [comment]
好的，我刚开始弄错变量了，现在可以了，谢谢
